javascript/statements.json

```diff
@@ -1499,6 +1487,84 @@
               }
             }
           }
+        },
+        "service_worker_support": {
+          "__compat": {
+            "description": "Available in service workers",
+            "support": {
+              "chrome": {
+                "version_added": "91"
+              },
+              "chrome_android": "mirror",
+              "deno": {
+                "version_added": false
```

Comment:
Note, I don't _think_ service workers make sense for node/deno, so set to false. Don't know if they can be deleted for this case.